### ISSUE TITLE
Unable to use Windows PowerShell ISE with NVDA

### ISSUE THREAD

**Issue by Emil-18** [nvaccess/nvda#15056]
### Steps to reproduce:
* Open Windows PowerShell ISE, type something and navigate the text with windows commands or the review cursor.
### Actual behavior:

NVDA will be silent or play error sounds. NVDA wil not report that you landed in the edit control when opening the program. The following is printed to the NVDA log if the logging level is set to debug warning.
DEBUGWARNING - scriptHandler.executeScript (17:09:50.702) - MainThread (10492):
error executing script: <bound method EditableText.script_caret_moveByCharacter of <NVDAObjects.Dynamic_WpfTextViewEditableTextWithAutoSelectDetectionUIA object at 0x07482890>> with gesture 'left arrow'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 289, in executeScript
  File "editableText.pyc", line 229, in script_caret_moveByCharacter
  File "editableText.pyc", line 164, in _caretMovementScriptHelper
  File "NVDAObjects\behaviors.pyc", line 220, in _caretScriptPostMovedHelper
  File "editableText.pyc", line 150, in _caretScriptPostMovedHelper
  File "speech\speech.pyc", line 1231, in speakTextInfo
  File "speech\types.pyc", line 42, in __iter__
  File "speech\speech.pyc", line 1273, in getTextInfoSpeech
  File "NVDAObjects\UIA\__init__.pyc", line 878, in getTextWithFields
  File "NVDAObjects\UIA\__init__.pyc", line 866, in _getTextWithFieldsForUIARange
  File "NVDAObjects\UIA\__init__.pyc", line 623, in _getTextWithFields_text
  File "NVDAObjects\UIA\__init__.pyc", line 231, in _getFormatFieldAtRange
  File "UIAHandler\utils.pyc", line 234, in __init__
  File "monkeyPatches\comtypesMonkeyPatches.pyc", line 32, in __call__
_ctypes.COMError: (-2147220991, 'An event was unable to invoke any of the subscribers', (None, None, None, 0, None))
### Expected behavior:

NVDA reports the control as normal.
### NVDA logs, crash dumps and other attachments:
[nvda.log](https://github.com/nvaccess/nvda/files/11871512/nvda.log)

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2023.1
#### Windows version:
Windows 10.0.19044 Build 19044
#### Name and version of other software in use when reproducing the issue:
powershell_ise 10.0.19041.1
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I have tried a 2016 version in the past. I can'tt remember which, and when I try to run any 2016 version now, it refuses to start on my system. The version I tried worked, as far as I can remember, fully with the main edit control 
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by seanbudd** [comment]
Welcome @Emil-18 ,

This is very likely a problem with the implementation of the application, as a starting point please submit feedback to Microsoft to let them know about the problem.

Kind Regards,
NV Access Software Developers

**Comment by seanbudd** [comment]
Given that this is an issue with ISE, and support for ISE has discontinued, I am closing this issue. It is unlikely MS will ever fix this.
